flink-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/JDBCOutputFormatTest.java

```diff
@@ -133,6 +133,32 @@ public void testExceptionOnInvalidType() throws IOException {
 		jdbcOutputFormat.writeRecord(row);
 	}
 
+	@Test(expected = RuntimeException.class)
```

Comment:
I am assuming you are trying to catch the `"imthewrongtype"` on the last column. but if the jdbc cast float to double fails. It will throw the same exception and the test will still pass. I suggest let this pass by setting the last field to `INTEGER`

Reply:
agree.